web-client/src/presenter/actions/getPDFForPreviewTabAction.ts

```diff
@@ -0,0 +1,21 @@
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const getPDFForPreviewTabAction = async ({
+  applicationContext,
+  get,
+  props,
+}: ActionProps) => {
+  if (props.file.name) {
+    return props;
+  }
+  const { docketEntryId } = props.file;
+  const docketNumber = get(state.caseDetail.docketNumber);
+
+  const pdfObj = await applicationContext
+    .getUseCases()
+    .loadPDFForPreviewInteractor(applicationContext, {
```

Comment:
What other error handling do you suggest for `loadPDFForPreviewInteractor` other than what's is already there because it is logging and then re-throwing the error. Also, I updated the return type to `Blob` to match the return type for `getDocument`